README.md

```diff
@@ -198,6 +198,20 @@ NAME                     DESIRED   READY   NODE SELECTOR   AGE
 gpudeviceplugin-sample   1         1                       5s
 ```
 
+In order to limit the deployment to a specific device type,
+use one of kustomizations under deployments/operator/device.
+
+For example, to limit the deployment to FPGA, use:
+
+```bash
+$ kubectl apply -k deployments/operator/device/fpga
+```
+
+Operator also supports deployments with multiple selected device types.
```

Comment:
Let's add a note that the CRDs are created but no controller is registered.

Also, please rebase. This section got moved to `cmd/operator/README`

Reply:
@mythi 

OK, thanks!